docs/src/test-annotations.md

```diff
@@ -124,3 +118,81 @@ Or if you want the opposite, you can skip the tests with a certain tag:
 ```bash
 npx playwright test --grep-invert @slow
 ```
+
+## Conditionally skip a group of tests
+
+For example, you can run a group of tests just in Chromium by passing a callback.
+
+```js js-flavor=js
+// example.spec.js
+
+test.describe('chromium only', () => {
+  test.skip(({ browserName }) => browserName !== 'chromium', 'Chromium only!');
+
+  test.beforeAll(async () => {
+    // This hook is only run in Chromium.
+  });
+
+  test('test 1', async ({ page }) => {
+    // This test is only run in Chromium.
+  });
+
+  test('test 2', async ({ page }) => {
+    // This test is only run in Chromium.
+  });
+});
+```
+
+```js js-flavor=ts
+// example.spec.ts
+
+test.describe('chromium only', () => {
+  test.skip(({ browserName }) => browserName !== 'chromium', 'Chromium only!');
+
+  test.beforeAll(async () => {
+    // This hook is only run in Chromium.
+  });
+
+  test('test 1', async ({ page }) => {
+    // This test is only run in Chromium.
+  });
+
+  test('test 2', async ({ page }) => {
+    // This test is only run in Chromium.
+  });
+});
+```
+
+## Use fixme in `beforeEach` hook
+
+To avoid running `beforeEach` hooks, you can put annotations in the hook itself.
+
+```js js-flavor=js
+// example.spec.js
+
+test.beforeEach(async ({ page, isMobile }) => {
+  test.fixme(isMobile, 'Settings page does not work in mobile yet');
```

Comment:
A user recently asked how to skip a test for mobile tests. I'm not a huge fan of isMobile since it does not support Firefox and recommended to him to skip it based on the project name via testInfo (he also asked for how to skip it based on projects).

Reply:
That seems fine, but I think `isMobile` is good as well.